eo-parser/src/main/resources/XMIR.xsd

```diff
@@ -113,6 +113,7 @@ SOFTWARE.
     <xs:attribute name="version" type="xs:string" use="required"/>
     <xs:attribute name="revision" type="xs:string" use="required"/>
     <xs:attribute name="dob" type="xs:dateTime" use="required"/>
+    <xs:attribute name="hash" type="xs:string" use="required"/>
```

Comment:
@Yanich96 Returning to the [discussion](https://github.com/objectionary/eo/pull/2771#discussion_r1450305042). Maybe we don't  need the `hash` attribute? Especially if we will save `hash` as a file attribute.

Reply:
@volodya-lombrozo 
I don't understand how to save hash as a file attribute.

Reply:
@Yanich96 You can always calculate the hash on the fly, like `hash(xml);`  and compare it with the cache file attribute.

Reply:
@volodya-lombrozo ok

Reply:
I have done todo. This line will be deleted after it.